README.md

```diff
@@ -11,36 +11,45 @@
 | Docs | [![gh-pages](https://img.shields.io/github/last-commit/holoviz/holoviz/gh-pages.svg)](https://github.com/holoviz/holoviz/tree/gh-pages) [![site](https://img.shields.io/website-up-down-green-red/http/holoviz.org.svg)](http://holoviz.org) [![dev-site](https://img.shields.io/website-up-down-green-red/https/pyviz-dev.github.io/holoviz.svg?label=dev%20website)](https://pyviz-dev.github.io/holoviz/)  |
 | Status Dashboard | [![](https://img.shields.io/website-up-down-green-red/http/status.holoviz.org.svg?label=status-dashboard)](http://status.holoviz.org/) |
 | Binder  | [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/holoviz/holoviz/HEAD?labpath=examples%2Ftutorial%2F00_Setup.ipynb) |
-| Code of Conduct | [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)|
+| Code of Conduct | [![Based on Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)|
 
 
-## What is it?
+## What is HoloViz?
 
-This repository provides examples, demos, and training materials
-documenting how to solve visualization problems using Python
-tools that work with web browsers, including
-[Bokeh](https://bokeh.org),
-[HoloViews](https://holoviews.org),
-[GeoViews](http://geoviews.org),
-[Matplotlib](https://matplotlib.org),
-[Datashader](https://datashader.org),
-[hvPlot](https://hvplot.holoviz.org),
-[Panel](https://panel.holoviz.org), and
-[Param](https://param.holoviz.org).
+HoloViz provides high-level tools that are designed to work together to solve the entire problem of visualization, from conducting exploratory data analysis to deploying complex dashboards.
 
-It includes a shared conda environment that ensures all these tools work together,
-facilities for downloading sample data for use with these tools, and provides
-starting points for solving your own visualization problems.
+The core HoloViz projects are:
 
+- [hvPlot](https://hvplot.holoviz.org): interactive plotting built on Bokeh, Matplotlib, and Plotly
+- [HoloViews](https://holoviews.org): declarative high-level data description and plotting
+- [GeoViews](http://geoviews.org): geographic data extension for HoloViews
+- [Datashader](https://datashader.org): server-side rendering of large datasets
+- [Colorcet](https://colorcet.holoviz.org/): perceptually based colormapping
+- [Panel](https://panel.holoviz.org): interactive apps and dashboards
+- [Lumen](https://lumen.holoviz.org/): no-code apps and dashboards
+- [Param](https://param.holoviz.org): declarative parameter-based coding
 
-## Installation
-
-See [holoviz.org](https://holoviz.org/installation.html) (or doc/installation.rst for a github clone).
-
-
-## About holoviz.org
-
-HoloViz is a project supported in part by [Anaconda](https://anaconda.com).
```

Comment:
Does it hurt to keep that?

Reply:
Personally, I don't know if it belongs in the readme. HoloViz could get support from a number of different sources and we don't want to convey the impression that you have to work at Anaconda in order to become a core maintainer. The upside to keeping it is that it could potentially convey to some people that we are relatively secure as an org, but I think there are better ways to communicate this aspect, such as commit/release frequency, good docs, diverse group of contributors, open meetings, and open governance.

Reply:
Alright thanks, convinced!